cdap-distributions/src/packer/cdap-sdk-ubuntu12-with-uri.json

```diff
@@ -6,7 +6,7 @@
     {
       "type": "virtualbox-iso",
       "guest_os_type": "Ubuntu_64",
-      "iso_url": "http://releases.ubuntu.com/12.04/ubuntu-12.04.5-server-amd64.iso",
+      "iso_url": "file:///Users/bamboo/Downloads/ubuntu-12.04.5-server-amd64.iso",
```

Comment:
Is this Mac OS-dependent code? Should that be tested for (the OS being run on) or noted in some way?